### PR TITLE
Sage/build a full stack coffee dapp with celo composer and solidity

### PR DESCRIPTION
**Pull request by gconnect**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/17-2-23-Build-a-Full-Stack-Coffee-Dapp-With-Celo-Composer-and-Solidity/Build-a-Full-Stack-Coffee-Dapp-With-Celo-Composer-and-Solidity.md [hunk 1]
```diff
@@ -0,0 +1,1004 @@
...
+description: This tutorial will take you through a step-by-step guide on how to create a frontend and backend (Smart Contract) dApp explaining how to create a decentralized version of Buy Me A Coffee.
+authors:
+  - name: Glory Agatevure
+    title: Blockchain Engineer | Technical Writer
+    url: https://www.linkedin.com/in/glory-agatevure-47a222ab/
+    image_url: https://avatars.githubusercontent.com/u/23031920?s=400&u=99aba40f495b86f3936a53e06dc7fbbc278b98ad&v=4
+tags: ['celosage','composer','celo','celowallet','reactcelo','contractkit','dapp','valora','typescript', 'smartcontract', 'solidity', 'nextjs', 'intermediate', 'advanced']
+hide_table_of_contents: true
+slug: /tutorials/Build-a-Full-Stack-Coffee-Dapp-With-Celo-Composer-and-Solidity
+
```
Comment (Ishanpathak1) [inline blog/17-2-23-Build-a-Full-Stack-Coffee-Dapp-With-Celo-Composer-and-Solidity/Build-a-Full-Stack-Coffee-Dapp-With-Celo-Composer-and-Solidity.md:12]: Add header image

File: src/data-tutorials/users.tsx [hunk 2]
```diff
@@ -866,6 +866,20 @@ const Users: User[] = [
     website: 'blog/tutorials/how-to-quickly-build-an-nft-collection-on-celo',
     tags: ['beginner','foundation', 'nft', 'foundation'],
   },
+  {
+    title: 'How should you use Celo Blockchain',
```
Comment (Ishanpathak1) [inline src/data-tutorials/users.tsx:870]: Can I know why it is changing two articles

**Review by Ishanpathak1 — COMMENTED** [review]
Make the users.tsx file correct and , add the header image

File: src/data-tutorials/users.tsx (hunk 2 above)
Comment (gconnect) [reply src/data-tutorials/users.tsx:870]: I initially had a git conflict there. And I notice it was pointing at those so I remove one that was duplicated

File: blog/17-2-23-Build-a-Full-Stack-Coffee-Dapp-With-Celo-Composer-and-Solidity/Build-a-Full-Stack-Coffee-Dapp-With-Celo-Composer-and-Solidity.md (hunk 1 above)
Comment (gconnect) [reply blog/17-2-23-Build-a-Full-Stack-Coffee-Dapp-With-Celo-Composer-and-Solidity/Build-a-Full-Stack-Coffee-Dapp-With-Celo-Composer-and-Solidity.md:12]: I have added the path to the header image on the users.tsx file.

Is there anything that is not correct on that file.

File: blog/17-2-23-Build-a-Full-Stack-Coffee-Dapp-With-Celo-Composer-and-Solidity/Build-a-Full-Stack-Coffee-Dapp-With-Celo-Composer-and-Solidity.md [hunk 3]
```diff
@@ -0,0 +1,1006 @@
...
+## Conclusion​
+Congratulations 🎉 on finishing this tutorial! Thank you for taking the time to complete it. In this tutorial, you have learned how to create a full stack dApp using Celo Composer and Solidity.
+
+To have access to the full codebase, here is the link to the project repo on [github](https://github.com/gconnect/Sage-buymeacoffee).
+
+## About the Author​
+Glory Agatevure is a blockchain engineer, technical writer, and co-founder of Africinnovate. You can connect with me on [Linkedin](https://www.linkedin.com/in/glory-agatevure-47a222ab/), [Twitter](https://twitter.com/agatevureglory) and [Github](https://github.com/gconnect/).
+
+## References​
+- https://github.com/celo-org/celo-composer
```
Comment (Ishanpathak1) [inline blog/17-2-23-Build-a-Full-Stack-Coffee-Dapp-With-Celo-Composer-and-Solidity/Build-a-Full-Stack-Coffee-Dapp-With-Celo-Composer-and-Solidity.md:997]: use for link [LINK](url)

**Review by viral-sangani — APPROVED** [review]

**Pull request merged**